### DISCUSSION TITLE
Set utility in templates

### DISCUSSION THREAD

**Discussion by DZ8540** [vueuse/vueuse#3085] [Q&A]
How can i use [set utility](https://vueuse.org/shared/set/) in templates?

```vue
<SomeComponent @click="set(someRef, someValue)" />
```

It throws error, because vue [unwrapped refs](https://vuejs.org/guide/essentials/reactivity-fundamentals.html#ref-unwrapping-in-templates) in templates.

**Comment by HamasakiBrain** [comment]
i have the same problem, up please
